src/main/java/com/faforever/client/remote/domain/RatingRange.java

```diff
@@ -1,37 +1,69 @@
 package com.faforever.client.remote.domain;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+import lombok.Value;
 import org.jetbrains.annotations.NotNull;
 
+import java.util.List;
+
+@Value
 public class RatingRange implements Comparable<RatingRange> {
 
-  private final Integer min;
-  private final Integer max;
+  Integer min;
+  Integer max;
 
   public RatingRange(Integer min, Integer max) {
     this.min = min;
     this.max = max;
   }
```

Comment:
Removed

Reply:
Actually this has to be present because the normal constructor is not produced by Value and it is needed to instantiate with a null as one of the values